frontend/packages/core/src/Resolver/index.tsx

```diff
@@ -117,30 +93,27 @@ const Resolver: React.FC<ResolverProps> = ({ type, searchLimit, onResolve, varia
         <Error message={state.schemaFetchError} onRetry={() => loadSchemas(type, dispatch)} />
       ) : (
         <Loadable variant="overlay" isLoading={state.resolverLoading}>
-          {process.env.REACT_APP_DEBUG_FORMS === "true" && <DevTool control={validation.control} />}
+          <CompressedError title="Error" message={state.resolverFetchError} />
```

Comment:
need mockups from designer, i think it does make sense though

if it's below the form it looks it belongs to either the query or schema resolver elements only

top applies to all

Reply:
sounds good. We can update this later but the width is off (or what I'd imagine is off) since it's max width is being restricted compared to the text field.

Reply:
per description, not worried about style in this PR, only logic since it's complicated enough on its own

Reply:
added comment on figma terminate instance for mockup of submit error